lib/twofishes/result.rb

```diff
@@ -16,41 +12,51 @@ def self.from_response(hash)
       end
     end
 
-    def name
-      feature.name
+    def what
+      @data['what']
     end
 
-    def display_name
-      feature.display_name
+    def where
```

Comment:
this changes the behavior compared to the implementation before (display_name is gone). also introduces a breaking change. accident or on purpose?